src/CoreFx.Private.TestUtilities/src/Resources/Strings.resx

```diff
@@ -0,0 +1,64 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
```

Comment:
This is a test library why can't it have strings hard coded?

Reply:
I wasn't sure. I'll hardcode it